doc/whats_new/v1.0.rst

```diff
@@ -400,6 +400,15 @@ Changelog
   :class:`ensemble.HistGradientBoostingRegressor`. Please use `tol=0` for
   the same behavior. :pr:`19296` by `Thomas Fan`_.
 
+- |API| Changed the default of :func:`max_features` to 1.0 for
```

Comment:
Yep as mentioned by @lorentzenchr we will need to move to 1.1. You will also need to update the version number here and elsewhere.